src/test/java/uk/gov/dhsc/htbhf/utils/NinoGenerator.java

```diff
@@ -12,8 +12,8 @@
     /**
      * Represents 1 child under 1 and 2 under 4 to make the value returned from the stub consistent
      */
-    private static final String CHILDREN_NUMBER = "12";
-    private static final String NINO_PREFIX_CHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+    private static final String CHILDREN_NUMBER = "11";
+    private static final String NINO_PREFIX_CHARS = "ABCDEFGHIJKLMNOPRSTUVWXYZ";
```

Comment:
If we're going to change this, shouldn't we also remove X as this is the failure NINO?